### ISSUE TITLE
Add option to strip text in brackets from imported deck titles

### ISSUE THREAD

**Issue by GaryKing** [HearthSim/Hearthstone-Deck-Tracker#2572]
Some decks, especially those from Hearthpwn, have unnecessary text in brackets at the start of deck titles, such as "[84% Winrate] Totem Shaman - Easy Climb", "[S26 Legend] Midrange + Guide!", and "[Legend streaming] Control".

I understand that some people wouldn't want that text removed, but perhaps add an option to strip that text from the titles if we want? Perhaps either a global switch, or a checkbox in the import dialog for each time we import a deck.

**Comment by tknights2015** [comment]
This would be great, as the full text is usually to long for HS

**Comment by GaryKing** [comment]
Yeah that was the reason I suggested this. The brackets often take up more space than the deck name's max length in HS. Of course I know I can remove these manually, but they are in about 50% of the top decks on Hearthpwn, and they rarely contain useful information, as you can see from my examples.
